.rubocop.yml

```diff
@@ -2,5 +2,14 @@ inherit_gem:
   rubocop-govuk:
     - config/default.yml
 
+inherit_mode:
+  merge:
+    - Exclude
+
+AllCops:
+  Exclude:
+    - "actions-gem-cache/**/*"
```

Comment:
Would it be possible to put these in a 'tmp' directory? This would mean they automatically get excluding during linting.

Reply:
That's an entirely better idea! Thanks, Ben!

Reply:
That worked! 🎉